src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -60,10 +60,17 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
   });
 
   if (__DEV__) {
-    warning(
-      browserSupportsResizeObserver && hasOverflow,
-      'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
-    );
+    if (hasOverflow) {
+      warning(
+        browserSupportsResizeObserver,
+        'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
+      );
+    } else {
```

Comment:
I don't think we need this warning... really we only need the warning if hasOverflow is set to true, and we don't have a ResizeObserver.  If hasOverflow is false but the browser doesn't has a resizeobserver, we don't care.

Reply:
implemented suggested change